packages/core/src/actions/smartAccount/internal/runMiddlewareStack.ts

```diff
@@ -46,12 +73,21 @@ export async function _runMiddlewareStack<
     throw new AccountNotFoundError();
   }
 
+  const { dummyPaymasterAndData, paymasterAndData } = {
+    dummyPaymasterAndData: bypassPaymasterAndData(overrides)
+      ? bypassPaymasterMiddleware
```

Comment:
```suggestion
      ? noopMiddleware
```

Reply:
we need to set `paymasterAndData` field v0.6 to `0x`. Noop just passthrough the struct.
I guess during resolveProperties, undefined field is set to '0x'

Reply:
yea setting to 0x should be handled already. otherwise this wouldn't have worked without this function before